app/models/user.rb

```diff
@@ -14,4 +16,21 @@ def User.digest string
       BCrypt::Engine.cost
     BCrypt::Password.create string, cost: cost
   end
+
+  def User.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
nếu đổi :remember_digest, User.digest(remember_token) =>>> remember_digest: User.digest(remember_token) thì nó báo lỗi "wrong number of arguments (given 1, expected 2)"